lib/Parse/ParseDecl.cpp

```diff
@@ -5794,6 +5794,13 @@ ParserStatus Parser::parseInheritance(
             TildeCopyableLoc = tildeLoc;
 
           continue;
+        } else if (nextTok.is(tok::code_complete)) {
+          consumeToken(); // consume '~'
+          consumeToken(); // consume code completion token
```

Comment:
```suggestion
          consumeToken(); // consume '~'
          consumeToken(tok::code_complete);
```
Also, I know we are not consistent about when to consume CC tokens. But the majority consumes it after the callback call.